.github/CONTRIBUTING.md

```diff
@@ -216,6 +216,26 @@ Instead of this you should;
 	do something different
 ````
 
+### Proc arguments
+Keep typed arguments consistent across redefinitions, based on what minimal type the proc is called with rather than the actual type of the passed argument.
+````DM
+/atom/proc/do_a_thing(mob/user)
+	do something
+
+/atom/movable/do_a_thing(mob/living/carbon/human/user)
+	do something
+````
+Instead of this you should do it like this:
+````DM
+/atom/proc/do_a_thing(mob/user)
+	do something
+
+/atom/movable/do_a_thing(mob/user)
+	if istype(user, mob/living/carbon/human)
+	typecast
```

Comment:
If you use the human example, then give a correct example here, indented.
Else do it wholly generic.